.github/workflows/dispatch-workflow.yml

```diff
@@ -11,9 +11,6 @@ on:
       wpt-args:
         required: true
         type: string
-      wpt-layout:
```

Comment:
We should not remove this one yet. This will require additional changes (we still want to enable & disable WPT)  that is going to be handled as part of #36047 .